### PR TITLE
Update for julia 1.4: add {Project,Manifest}.toml, rm require

### PR DESCRIPTION
**Pull request by greimel**
tests pass locally

### CODE REVIEW

**Comment by oyamad** [comment]
@greimel Many thanks!

Just one question, do we need to include `Manifest.toml` files?

**Comment by greimel** [comment]
They are not necessary. Opinions differ, see https://discourse.julialang.org/t/does-manifest-toml-belong-in-the-repository/12029

Shall I remove them?

**Comment by oyamad** [comment]
Yes please, thanks!

**Comment by greimel** [comment]
done

**Comment by greimel** [comment]
Please add the email address related to your github account (or remove the stub) to Project.toml before you merge. Thanks!

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name = "ContinuousDPs"
+uuid = "c470661a-f883-5a79-ade0-a473a39194e2"
+authors = ["Daisuke Oyama <email@address.com>"]
```
Comment (greimel) [inline Project.toml:3]: please adjust this line!

**Pull request merged**